pkg/sysfs/system.go

```diff
@@ -664,6 +678,12 @@ func (c *cpu) Isolated() bool {
 	return c.isolated
 }
 
+// SstClos returns the Speed Select Core Ppwer CLOS number assigned to the CPU
```

Comment:
typo: still one more typo on that line... s/Ppwer/Power/

Reply:
Thanks. Fixed